userspace/libpman/src/state.c

```diff
@@ -25,7 +25,7 @@ limitations under the License.
 
 struct internal_state g_state = {};
 
-static void log_error(const char* fmt, ...)
+static void log_msg(enum falcosecurity_log_severity level, const char* fmt, ...)
```

Comment:
we need to use the `level` inside this function, atm it's not used

Reply:
/me facepalming

Sure, give me a moment.